src/sdk-reference/js/6/realtime/subscribe/snippets/user-notifications.js

```diff
@@ -1,28 +1,13 @@
 function callback (notification) {
   if (notification.type === 'user') {
-    console.log(notification);
+    console.log(notification.volatile);
     /*
-    { status: 200,
-      timestamp: 1539696822690,
```

Comment:
It's better to have the full notification content so users have a visual representation of the object received